lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -333,6 +333,13 @@
       - List of the service networks names.
       - Corresponds to the C(--network) option of C(docker service create).
     type: list
+  stop_grace_period:
+    description:
+        - Time to wait before force killing a container.
+        - "Accepts a duration as a string in a format that look like:
+          C(5h34m56s), C(1m30s) etc. The supported units are C(us), C(ms), C(s), C(m) and C(h)."
+        - Corresponds to the C(--stop-grace-period) option of C(docker service create).
+    type: str
```

Comment:
There needs to be a `version_added` here (after this line). Besides that, looks good!